### ISSUE TITLE
global vars can be editted / broken

### ISSUE THREAD

**Issue by ChrisJefferson** [gap-system/gap#813]
the function `IDENTS_GVAR`, used by `NamesSystemGVars`, gives out references to strings which GAP uses to represent global variable names. We probably just shouldn't make these strings user-accessable, as breaking them completely breaks GAP.

Canonical example:

```
gap> Filtered(NamesSystemGVars(),name->IsSubset(LETTERS,name));;
gap> X;
Error, Variable: 'X' must have a value
not in any function at *stdin*:2
gap> Z;
Error, Variable: 'Z' must have a value
not in any function at *stdin*:3
gap> Sort;
Error, Variable: 'Sort' must have a value
not in any function at *stdin*:4
gap> List;
Error, Variable: 'List' must have a value
not in any function at *stdin*:5
```

**Comment by ChrisJefferson** [comment]
Fixed in #814

**Comment by fingolfin** [comment]
Resolved by merging #814
